src/transformState.js

```diff
@@ -46,7 +46,29 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
+  for (const object of transforms) {
+    const properties = object.properties;
```

Comment:
```suggestion
    const { properties } = object;
```
https://learn.javascript.ru/destructuring